plugins/obs-webrtc/whip-output.cpp

```diff
@@ -279,6 +284,57 @@ bool WHIPOutput::Setup()
 	return true;
 }
 
+void WHIPOutput::ParseLinkHeader(std::string val,
```

Comment:
Might be nice to put some of the example strings in documentation for the method.

Reply:
Done!